changelog.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## What's New (02/18/2020)
+
+### New validation rules
+- OperationIdRequired rule - each operation must has a unique operationId.
```

Comment:
must have

Reply:
thanks, could you please review this again, I have modified it